sasmodels/models/bcc_paracrystal.py

```diff
@@ -18,47 +13,105 @@
 
 .. math::
+    \vec{a_1} &= \frac{1}{2}(-\vec{b_1} + \vec{b_2} + \vec{b_3}) \\
+    \vec{a_2} &= \frac{1}{2} (\vec{b_1} - \vec{b_2} + \vec{b_3}) \\
+    \vec{a_3} &= \frac{1}{2}(\vec{b_1} + \vec{b_2} -\vec{b_3}).
 
-    V_\text{lattice} = \frac{16\pi}{3} \frac{R^3}{\left(D\sqrt{2}\right)^3}
+where $\vec{b_1},\vec{b_2}$, and $\vec{b_3}$ are the unit cell vectors of the
+conventional unit cell, which is a unit cell that includes the full symmetry
+of the lattice. As defined by reference [#Matsuoka1987]_, the constant $a$ is the
+lattice parameter of the conventional unit cell with
+$|\vec{b_1}|=|\vec{b_2}|=|\vec{b_3}|=a$. Using this definition, the
+nearest-neighbor distance ($D$) is given by
+$D=|\vec{a_1}|=|\vec{a_2}|=|\vec{a_3}|=\sqrt{(a/2)^2+(a/2)^2+(a/2)^2}=\sqrt{\frac{3a^2}{4}}=\frac{\sqrt{3}a}{2}$.
 
+The volume of the primitive unit cell $V_p$ is then given by:
 
-The distortion factor (one standard deviation) of the paracrystal is included
-in the calculation of $Z(q)$
+.. math::
+    V_p &= |(\vec{a_1}\times \vec{a_2})\cdot\vec{a_3}|\\
+    &= (\frac{a^2}{2},\frac{a^2}{2},0)\cdot(\frac{a}{2},\frac{a}{2},-\frac{a}{2})\\
+    &= a^3/2
+
+In this case, the volume fraction ($V_{latt}$) of spherical particles with
+radius $R$ sitting on the bcc lattice is given by:
 
 .. math::
+    V_{latt} &= \frac{4/3 \pi R^3}{a^3/2}\\
+    &= \frac{8\pi R^3}{3a^3}\\
+    &= \frac{\sqrt{3} \pi R^3}{D^3}
 
-    \Delta a = g D
+Now, continuing to follow [#Matsuoka1987]_, we define the structure(lattice)
+factor $Z(\vec{q})$, which is related to the scattering intensity by
+$I(\vec{q})=N |f_0|^2 Z(\vec{q})$ where N is the number of particles in the bcc
+crystal, and $\vec{q}$ is the momentum transfer. For a 3D paracrystal:
 
-where $g$ is a fractional distortion based on the nearest neighbor distance.
+.. math::
+    Z(\vec{q}) = \prod_{k=1}^{3}Z_k(\vec{q})
 
+with
 
-.. figure:: img/bcc_geometry.jpg
+.. math::
+    Z_k(\vec{q}) = \frac{1-|F_k|^2}{1-2|F_k|cos(\vec{a_k}\cdot\vec{q})+|F_k|^2}
 
-    Body-centered cubic lattice.
+and where $F_k(\vec{q})$ is the structure factor of the primitive unit cell
+defined as:
 
-For a crystal, diffraction peaks appear at reduced q-values given by
+.. math::
+    F_k(\vec{q}) = e^{-\frac{1}{2} \Delta a^2_k q^2} \times e^{-i\vec{q}\cdot\vec{a_k}}.
+
+Here, $\vec{a_k}$ are the primitive unit cell vectors $\vec{a_1}$, $\vec{a_2}$,
+and $\vec{a_3}$. Furthermore, $\Delta a_k$ is the isotropic distortion of the
+lattice point from its ideal position and can be defined by a constant factor
+$g=\Delta a / |\vec{a_1}| = \Delta a / |\vec{a_2}| = \Delta a / |\vec{a_3}|=\Delta a/D$.
+
+Finally, assuming the definitions presented in this document, the authors of
+reference [#Matsuoka1987]_ have derived the lattice factors which are given by:
 
 .. math::
+    Z_1(q,\theta,\phi)&=[1-e^{-q^2\Delta a^2}]/\{1-2e^{-\frac{1}{2}q^2\Delta a^2}cos[\frac{qa}{2}(sin\theta cos\phi + sin\theta sin\phi + cos\theta)] + e^{-q^2\Delta a^2}\}\\
```

Comment:
The 'sincos' and 'sinsin' in these definitions of Z1, Z2 & Z3 look ugly. Maybe separate with spaces?

Reply:
That is standard latex output which I'm not sure how to override.  Figure we can let the latex snobs decide what spacing looks "proper"  :-)

Reply:
Would expect the trig functions to be `\sin` and `\cos` in there. Without that, they get rendered as _s_ multiplied by _i_ multiplied by _n_, which certainly is ugly.

Reply:
oops!  yeah-- thanks @llimeht